draft-scheffenegger-congress-rfc5033bis.md

```diff
@@ -292,6 +291,40 @@ trip times comparable bandwidth during backoff.
 alternate congestion control algorithm, the proposal should
 explore how bandwidth is shared among the competing flows.
 
+(*)
+: Utilising More than one Path.
+
+: An IETF transport is not tied to a specific Internet path.
+The set of routers forming the path can and do change with time,
+this will also cause the properties of the path to change with respect to time.
+Congestion control needs to be designed to take into account
+these changes in the path characteristics.
+Some transports permit more than one path to be differentiated and used by
+a single connection at the sender. These are called multipath transport protocols.  
+A multipath sender can schedule which packets travel on which of its active paths.
+This enables a tradeoff in timeliness and reliability.
+
+One use is to provide fail-over from one path to
+another when the original path is no longer viable or to switch the traffic from
+one path to another when this is expected to improve performance
+(latency, throughput, reliability, cost).
+A design independently needs to track the congestion state of each path,
+and needs to demonstrate independent congestion control for each path being used.
+
+: Anothe use is for concurrent multipath scheduling. 
```

Comment:
I do think we can find/have consensus on some wording around concurrent multipath. (We owe it to hint why this has been on the edge of what the IETF has explored - SCTP CMT since ~2010, and other research which the IETF has not adopted.) WG chairs and AD's have consistently been wary of concurrent multipath, so we ought to indicate something, the last sentence is such as “current” statement.

Reply:
Also, picking-up on your comment on "unfair" design, I agree. That was not where I saw the additional concerns for CMT: These were about avoiding harm/starvation and achieving stability. I originally used "appropriate”, only because I thought we would define that earlier, but I would be happy enough to say harm (or starvation) providing this is defined earlier. I’d love to add consideration of stability with respect to shifting traffic between multiple paths in a competitive way, if there is agreement, but have not done so yet, because that’s one stage deeper...

Reply:
Thought resolved